documentation/asciidoc/computers/configuration/device-tree.adoc

```diff
@@ -404,7 +404,7 @@ Assignments appear at the end of a declaration, and are indicated by a `=`:
 [source,kotlin]
 ----
 str_val  = <&target>,"strprop=value";              // 1
-int_val  = <&target>,"intprop:0=42                 // 2
+int_val  = <&target>,"intprop:0=42"                // 2
```

Comment:
I am _reasonably_ sure that this was a typo, originally, Will keep an eye out to see if this gets reported as an issue any time soon!